nanoFirmwareFlasher.Library/FirmwarePackage.cs

```diff
@@ -26,28 +23,31 @@ namespace nanoFramework.Tools.FirmwareFlasher
     public abstract class FirmwarePackage : IDisposable
     {
         // HttpClient is intended to be instantiated once per application, rather than per-use.
-        static readonly HttpClient _cloudsmithClient;
+        static readonly HttpClient s_cloudsmithClient;
 
-        private const string _refTargetsDevRepo = "nanoframework-images-dev";
-        private const string _refTargetsStableRepo = "nanoframework-images";
-        private const string _communityTargetsRepo = "nanoframework-images-community-targets";
+        private const string RefTargetsDevRepo = "nanoframework-images-dev";
```

Comment:
Code style call for small caps on privates, starting with underscore.

Reply:
Actually no, that is not what your .editorconfig rules say. That is: you are right about fields, but the .editorconfig rules do not consider `private const` a static field (and have a name starting with `s_smallcaps`). For constants the rule is that they should start with a capital.

I don't mind having it one way or the other, but it would be nice to keep the .editorconfig and your coding style in sync :-)

Reply:
Then this is probably a deviation that we've been using on our code base...
If we are to have it all in sync (which we should as we're adhering to this) then all repos have to be updated and all these renamed accordingly.

Reply:
Let's go with the editorconfig rule then. That's why we've added it in the first place.